readFromPdf.py

```diff
@@ -1,9 +1,144 @@
 import os
+import re
 import argparse
 import subprocess
 
 from src.chunker import try_to_write_dir
 
+def pdfToTxt(pdfname):
+    #creating a txt output file
+    txtname = pdfname[:-4]
+    txtname += ".txt"
+
+    # creating a pdf file object
+    pdfFileObj = open(pdfname, 'rb')
+    
+    # creating a pdf reader object
+    pdfReader = PyPDF2.PdfReader(pdfFileObj)
+
+    # get total pages of pdf
+    totalPages = len(pdfReader.pages)
+    fileText = ''
+
+    for i in range(totalPages):
+        # creating a page object
+        pageObj = pdfReader.pages[i]
+
+        # extracting text from page
+        pageText = pageObj.extract_text()
+
+        #filter the junk text
+        pageText = filterPageText(pageText) 
+        
+        #write the filtered text into a txt file
+        fileText += pageText
+
+    # closing the pdf file object
+    pdfFileObj.close()
+
+    #write the text from pdf to txt file
+    output = open(txtname, 'w')
+    output.write(fileText)
+
+
+#An ganze txt-Datei anwenden
+def filterPageText(pageText):
```

Comment:
Is the argument a line or a page of the document?
Name variables consistent. Please write the comments in English.